config/fields.yml

```
@@ -1230,6 +1230,15 @@
   searchable: true
   sortable: false
 
+- id: callnumber_browse
+  metadata:
+    name: Call Numbers
+    short_desc: Call numbers
```

```suggestion
    name: Call Number
    short_desc: Call Number
```
When grabbing the data, the singular and plural versions are the same:
```yml
term: "Call Numbers"
termPlural: "Call Numbers"
```